projects/website/markdown/docs/plugins/creating-plugins.mdx

```diff
@@ -56,6 +56,57 @@ In order for Tuist to locate the templates for a plugin, they must be placed in
 └── ...
 ```
 
+## ResourceSynthesizers
+
+ResourceSynthesizer plugins are for sharing & reusing templates for synthesizing resources. If you want to use one of the predefined resource synthesizers, the template must also adhere to a specific naming.
```

Comment:
```suggestion
ResourceSynthesizer plugins are for sharing & reusing templates for [synthesizing resources](/docs/usage/resources/). If you want to use one of the predefined resource synthesizers, the template must also adhere to a specific naming.
```

Reply:
Add another sentence after this one with a link to the Resource synthesizer project description docs 🙏🏼